src/cosalib/cmdlib.py

```diff
@@ -575,3 +575,29 @@ def ensure_glob(pathname, **kwargs):
 def ncpu():
     '''Return the number of usable CPUs we have for parallelism.'''
     return int(subprocess.check_output(['kola', 'ncpu']))
+
+
+def get_locked_nevras(srcdir, arch=None):
```

Comment:
Hmm, not sure I understand why we still have this. There's no need for merge semantics here I don't think. And at that point, the caller can just load the base lockfile directly.

Reply:
we still want overrides for pins don't we ? 
If i understand your comment it means the lockfile already include those after `cosa fetch --update-lockfile` runs ? In which case yeah, we don't we need this

Reply:
Yes, sorry. I was skimming through comments here and did not fully load the context. It's now fixed